src/app/main/component/comments/services/comments.service.spec.ts

```diff
@@ -0,0 +1,210 @@
+import { TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { FormControl, FormGroup } from '@angular/forms';
+import { CommentsService } from './comments.service';
+import { HttpResponse } from '@angular/common/http';
+
+describe('CommentsService', () => {
+  let service: CommentsService;
+  let httpTestingController: HttpTestingController;
+
+  const form = new FormGroup({
+    content: new FormControl('some')
+  });
+
+  beforeEach(() =>
+    TestBed.configureTestingModule({
+      imports: [HttpClientTestingModule],
+      providers: [CommentsService]
+    })
+  );
+
+  beforeEach(() => {
+    service = TestBed.get(CommentsService);
+    httpTestingController = TestBed.get(HttpTestingController);
+  });
+
+  afterEach(() => {
+    httpTestingController.verify();
+  });
+
+  it('should be created', () => {
+    const serviceNew: CommentsService = TestBed.get(CommentsService);
+    expect(serviceNew).toBeTruthy();
+  });
+
+  it('should change ecoNewId value', () => {
+    service.ecoNewsId = '1';
+    expect(service.ecoNewsId).toEqual('1');
+  });
+
+  it('should make POST request to add comment', () => {
+    const commentBody: object = {
```

Comment:
It's better to use `any` in such places in unit tests because `object` is more restrictive than `any`
Read this for more info (https://stackoverflow.com/a/18961904)
Please change it in all the places